all_channels_test.go

```diff
@@ -18,36 +18,41 @@
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 // THE SOFTWARE.
 
-package tchannel
+package tchannel_test
```

Comment:
Are these changes only for the `Sleep` after closing the channels?

If we need to keep this, can we dot import "tchannel-go" like in other tests, so that the test code itself doesn't change.